### PR TITLE
Simplify iterating over Element attributes

### PR DESCRIPTION
**Pull request by frewsxcv**
[Review on Reviewable](https://reviewable.io/reviews/servo/servo/8459)

### CODE REVIEW

**Comment by eefriedman** [comment]
Why not just use Element::attrs()?

**Comment by frewsxcv** [comment]
> Why not just use Element::attrs()?

Because that would be too obvious.

File: components/script/dom/element.rs [hunk 1]
```diff
@@ -785,9 +780,7 @@ impl Element {
                     }
 
                     // Step 2.
-                    let attrs = element.Attributes();
-                    for i in 0..attrs.Length() {
-                        let attr = attrs.Item(i).unwrap();
+                    for attr in self.attrs.borrow().iter() {
```
Comment (frewsxcv) [inline components/script/dom/element.rs]: @eefriedman This is wrong. Fixed in the latest force push.

**Comment by jdm** [comment]
@bors-servo: retry
* #8468

**Pull request merged**